assets/src/edit-story/elements/text/output.js

```diff
@@ -46,8 +46,8 @@ function TextOutput({
     fontFamily: generateFontFamily(fontFamily, fontFallback),
     fontWeight: fontWeight ? fontWeight : null,
     background: backgroundColor,
-    color:color,
-    lineHeight:lineHeight,
+    color,
+    lineHeight,
```

Comment:
@barklund I really cannot reproduce these issues with object shorthand syntax. Can you please help me understand that better?

* How did you set up prettier / eslint in your IDE of choice
* What happens when you run `npx prettier assets/src/edit-story/elements/text/output.js --write`
* What happens when you run `npm run lint:js assets/src/edit-story/elements/text/output.js`

Reply:
It's not an IDE thing - it's the husky hook, that does it.

It's specifically `lint:css:fix` that's the problem. And not even `lint:css` reports it - it's only the autofixer that does it!

If I run `npm run lint:css:fix assets/src/edit-story/elements/text/output.js`, I will get this exact change in this file.

Reply:
Haha, just discovered that `npm run lint:css:fix` breaks it, but `npm run lint:js:fix` re-fixes it. So it's basically a race condition on those two processes as to which finishes first - and if the js one finishes first, the file is left "damaged" by the CSS "fixer".

Reply:
Interesting. I cannot reproduce this behavior with `npm run lint:css:fix assets/src/edit-story/elements/text/out` though.

Sure, `npm run lint:js:fix` changes it to a shorthand notation, but `npm run lint:css:fix` doesn't change anything, no matter what I do with this style object.

Does this still happen even after a rebase and a fresh `npm install`?

Reply:
After rebase and re-install it doesn't seem to err anymore, so thanks!